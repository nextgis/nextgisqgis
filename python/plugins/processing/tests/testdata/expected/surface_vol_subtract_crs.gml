<?xml version="1.0" encoding="utf-8" ?>
<ogr:FeatureCollection
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://ogr.maptools.org/ surface_vol_subtract_crs.xsd"
     xmlns:ogr="http://ogr.maptools.org/"
     xmlns:gml="http://www.opengis.net/gml">
  <gml:boundedBy><gml:null>missing</gml:null></gml:boundedBy>
                                                                                                                                                                                                                                                                                                
  <gml:featureMember>
    <ogr:surface_vol_subtract_crs fid="surface_vol_subtract_crs.0">
      <ogr:volume>-65866966.86906432</ogr:volume>
      <ogr:m2>23202698.77333090</ogr:m2>
      <ogr:pixel_count>130550</ogr:pixel_count>
    </ogr:surface_vol_subtract_crs>
  </gml:featureMember>
</ogr:FeatureCollection>
