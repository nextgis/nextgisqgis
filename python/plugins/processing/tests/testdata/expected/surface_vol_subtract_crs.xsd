<?xml version="1.0" encoding="UTF-8"?>
<xs:schema targetNamespace="http://ogr.maptools.org/" xmlns:ogr="http://ogr.maptools.org/" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:gml="http://www.opengis.net/gml" elementFormDefault="qualified" version="1.0">
<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/feature.xsd"/>
<xs:element name="FeatureCollection" type="ogr:FeatureCollectionType" substitutionGroup="gml:_FeatureCollection"/>
<xs:complexType name="FeatureCollectionType">
  <xs:complexContent>
    <xs:extension base="gml:AbstractFeatureCollectionType">
      <xs:attribute name="lockId" type="xs:string" use="optional"/>
      <xs:attribute name="scope" type="xs:string" use="optional"/>
    </xs:extension>
  </xs:complexContent>
</xs:complexType>
<xs:element name="surface_vol_subtract_crs" type="ogr:surface_vol_subtract_crs_Type" substitutionGroup="gml:_Feature"/>
<xs:complexType name="surface_vol_subtract_crs_Type">
  <xs:complexContent>
    <xs:extension base="gml:AbstractFeatureType">
      <xs:sequence>
        <xs:element name="volume" nillable="true" minOccurs="0" maxOccurs="1">
          <xs:simpleType>
            <xs:restriction base="xs:decimal">
              <xs:totalDigits value="21"/>
              <xs:fractionDigits value="8"/>
            </xs:restriction>
          </xs:simpleType>
        </xs:element>
        <xs:element name="m2" nillable="true" minOccurs="0" maxOccurs="1">
          <xs:simpleType>
            <xs:restriction base="xs:decimal">
              <xs:totalDigits value="21"/>
              <xs:fractionDigits value="8"/>
            </xs:restriction>
          </xs:simpleType>
        </xs:element>
        <xs:element name="pixel_count" nillable="true" minOccurs="0" maxOccurs="1">
          <xs:simpleType>
            <xs:restriction base="xs:long">
              <xs:totalDigits value="20"/>
            </xs:restriction>
          </xs:simpleType>
        </xs:element>
      </xs:sequence>
    </xs:extension>
  </xs:complexContent>
</xs:complexType>
</xs:schema>
