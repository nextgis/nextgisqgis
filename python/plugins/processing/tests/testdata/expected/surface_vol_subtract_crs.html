<html><head><meta http-equiv="Content-Type" content="text/html;charset=utf-8"/></head><body>
<p>Analyzed file: /home/nyall/dev/QGIS/python/plugins/processing/tests/testdata/custom/dem_crs.tif (band 1)</p>
<p>Volume: -65866966.86906432</p>
<p>Pixel count: 130550</p>
<p>Area: 23202698.7733309 m2</p>
</body></html>